components/resourceDetails/resourceSharing/addAgentButton/agentPickerModal/index.test.jsx

```diff
@@ -29,18 +29,24 @@ import AgentPickerModal, {
   handleSubmit,
 } from "./index";
 import * as AddressBookFns from "../../../../../src/addressBook";
+import * as personModelFunctions from "../../../../../src/models/contact/person";
 import { renderWithTheme } from "../../../../../__testUtils/withTheme";
 import mockAccessControl from "../../../../../__testUtils/mockAccessControl";
-import mockPersonContactThing from "../../../../../__testUtils/mockPersonContactThing";
-
+import mockPersonContact from "../../../../../__testUtils/mockPersonContact";
+import mockGroupContact from "../../../../../__testUtils/mockGroupContact";
+import mockAddressBook from "../../../../../__testUtils/mockAddressBook";
 import AccessControlContext from "../../../../../src/contexts/accessControlContext";
 import * as ProfileFns from "../../../../../src/solidClientHelpers/profile";
 import ConfirmationDialogContext from "../../../../../src/contexts/confirmationDialogContext";
 import useNamedPolicyPermissions from "../../../../../src/hooks/useNamedPolicyPermissions";
 import usePermissionsWithProfiles from "../../../../../src/hooks/usePermissionsWithProfiles";
+import useAddressBook from "../../../../../src/hooks/useAddressBook";
+import useContacts from "../../../../../src/hooks/useContacts";
 
+jest.mock("../../../../../src/hooks/useAddressBook");
```

Comment:
Let's group the corresponding mocked instances, e.g. `const mockedAddressBookHook = useAddressBook;` in this case.